tests/views/test_scores.py

```diff
@@ -85,7 +85,7 @@ def setUpTestData(cls):
         cls.chosen_internship = InternshipFactory(cohort=cls.cohort, speciality=None)
         internships = [cls.mandatory_internship, cls.long_internship, cls.chosen_internship]
         periods = [PeriodFactory(cohort=cls.cohort) for _ in range(2)]
-        periods.append(cls.period)
+        periods.insert(0, cls.period)
```

Comment:
periods = [cls.period] + [PeriodFactory(cohort=cls.cohort) for _ in range(2)]
J'sais pas si c'est mieux que l'insert?